### PR TITLE
Adding dataTableVariable property to Services/DataTables

### PR DESCRIPTION
**Pull request by lk77**
Hello,

i'm adding dataTableVariable property to Services/DataTables.
The default value is dataTable like before, but can be overwritten by setting the dataTableVariable property in our custom dataTable class :

```
$dataTableVariable = 'MyDataTable';
```

Thanks.

### CODE REVIEW

File: src/Services/DataTable.php [hunk 1]
```diff
@@ -18,7 +18,14 @@
      * @var string
      */
     protected $printPreview = 'datatables::print';
-
+    
+    /**
+     * Name of the dataTable variable
```
Comment (yajra) [inline src/Services/DataTable.php]: Please add a period.

**Comment by yajra** [comment]
Looks good. Just need some patch on docs. Thanks!

**Comment by lk77** [comment]
I've corrected the typo. 
For the docs, i don't know where to put it, in buttons-starter.md ?

**Comment by yajra** [comment]
Thanks, maybe we should add a new doc file to document service class properties? There are also undocumented properties like exportColumns, printColumns, printPreview, etc..

**Comment by lk77** [comment]
yes i think we need a new documentation file for the properties and methods of the service class

**Pull request merged**